ballerina/Ballerina.toml

```diff
@@ -1,19 +1,19 @@
 [package]
 org = "ballerina"
 name = "xmldata"
-version = "2.8.1"
+version = "2.9.0"
 authors = ["Ballerina"]
 keywords = ["xml", "json"]
 repository = "https://github.com/ballerina-platform/module-ballerina-xmldata"
 icon = "icon.png"
 license = ["Apache-2.0"]
-distribution = "2201.10.0"
+distribution = "2201.11.0"
```

Comment:
Have we migrate this with Update 11(Java 21)? I thought this package is deprecated

Reply:
We considered upgrading all deprecated libraries to update 11 (Java 21) because users relying on deprecated libraries cannot upgrade to Java 11.